cabal-install/src/Distribution/Client/ProjectConfig/Legacy.hs

```diff
@@ -1408,7 +1408,8 @@ legacySharedConfigFieldDescrs constraintSrc =
               configPackageDBs
               (\v conf -> conf{configPackageDBs = v})
           ]
-        . filterFields (["verbose", "builddir"] ++ map optionName installDirsOptions)
+        . aliasField "keep-temp-files" "haddock-keep-temp-files"
```

Comment:
What is this line doing?

Reply:
Oh, this line shouldn't be here. It's so that when the `cabal.project` parser sees `haddock-keep-temp-files`, it uses the new `keep-temp-files` field, but I don't actually add that until #10292.